### PR TITLE
Interactive Indicator Correlation Section

### PR DESCRIPTION
**Pull request by dlaliberte**
closes 

**Prerequisites**:

- [ ] Unless it is a hotfix it should be merged against the `dev` branch
- [ ] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [ ] Build is successful
- [ ] Code is cleaned up and formatted

### CODE REVIEW

**Comment by dlaliberte** [comment]
image

**Comment by tildechris** [comment]
image

* What are the numbers (121) and (120)?  Can we delete this?  This information should be in the chart title.
* Chart title should be "X Indicator correlated with Y lagged by Z days"
* Y-axis indicator should be the context indicator.

Finally we need a mobile layout strategy:

image

**Comment by dlaliberte** [comment]
> image
> 
> * What are the numbers (121) and (120)?  Can we delete this?  This information should be in the chart title.

Leftover from Sam's template.... shows that the data has already been loaded.

> * Chart title should be "X Indicator correlated with Y lagged by Z days"

OK.

> * Y-axis indicator should be the context indicator.

OK.

File: src/stores/questions.raw.txt [hunk 1]
```diff
@@ -242,7 +242,7 @@ LearnMoreLink: https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals
 AddedInWave: 10
 OldRevisions:
  - Change: This version asked about the past 5 days, rather than 7. This was changed in Wave 10 to be consistent with other items on the survey.
-   Signal: smoothed_felt_isolated_7d
```
Comment (tildechris) [inline src/stores/questions.raw.txt:245]: Is this a merge error?

Comment (tildechris) [reply src/stores/questions.raw.txt:245]: Nevermind, this is from #871

**Comment by dlaliberte** [comment]
image

**Comment by tildechris** [comment]
For the correlation table, I think having the snake plot in there is too small to be helpful.  Sam had the idea that we could show a sparkline of the lag vs. r2 plot there.  Thoughts?

**Comment by dlaliberte** [comment]
> For the correlation table, I think having the snake plot in there is too small to be helpful. Sam had the idea that we could show a sparkline of the lag vs. r2 plot there. Thoughts?

Sure, but a sparkline of lag vs r2 for each indicator pair would definitely require precomputation of all those r^2 values, as far as I know.  I was confused by what was intended for the correlation table, so now I'm not sure I can help much more without the precomputed input data.

**Comment by dlaliberte** [comment]
image

**Pull request closed without merging**